a5/core/utils.py

```diff
@@ -0,0 +1,137 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) A5 contributors
+
+import numpy as np
+from typing import List, Tuple, Optional, TypedDict
+from .origin import Origin
+from .triangle import Triangle
+from .coordinate_systems import Degrees, LonLat, Face  
+
+vec2 = np.array
+vec3 = np.array
+mat2 = np.ndarray
+mat2d = np.ndarray
+
+Pentagon = List[Face]
+Contour = List[LonLat]
+
+class PentagonShape:
```

Comment:
I moved the `Origin` type to `utils.ts` to avoid a circular dependency: https://github.com/felixpalmer/a5/pull/20

Could you do the same in Python?